utilities_common/helper.py

```diff
@@ -0,0 +1,64 @@
+from dump.match_infra import MatchEngine, MatchRequest, ConnectionPool
+from dump.match_helper import get_matched_keys
+from .db import Db
+from .constants import DEFAULT_NAMESPACE
+
+def get_port_acl_binding(db_wrap, port):
+    """
+    Verify if the port is not bound to any ACL Table
+    
+    Args:
+        db_wrap: utilities_common.Db() object
+        port: Iface name
+    
+    Returns:
+        list: ACL_TABLE names if found, 
+                otherwise empty
+    """ 
+    ACL = "ACL_TABLE" # Table to look for port bindings
+    if not isinstance(db_wrap, Db):
+        raise Exception("db_wrap object is not of type utilities_common.Db")
+
+    conn_pool = ConnectionPool()
+    conn_pool.fill(DEFAULT_NAMESPACE, db_wrap.db, db_wrap.db_list)
```

Comment:
@vivekreddynv looks like `multiasic` case is not covered here

Reply:
So, does PBH_TABLE/ACL_TABLE sits it its own namespace and not in the DEFAULT namespace?

Reply:
@vivekreddynv if i am not mistaken config db is per asic, but suggest to double check

Reply:
As i understand, not everything in the config db is split b/w multi asic. I've inferred this from the config/plugins/pbh.py code. You're using Db.db object for redis queries and that connects to default namespace and so i thought PBH sits in default namespace.

Reply:
@vivekreddynv this is how it was implemented, still having some concerns here. That was a motivation for asking to double check.

Reply:
I've currently moved both the tables to use the same namespace as the one iface is present. I've confirmed the case for ACL table. Would need your help is determining PBH.